testgrid/config/config.yaml

```diff
@@ -1099,6 +1099,14 @@ dashboards:
     test_group_name: kubernetes-e2e-gce-gci-qa-slow-m57
   - name: qa-slow-master
     test_group_name: kubernetes-e2e-gce-gci-qa-slow-master
+# END google-gci-dev
+
+- name: google-gci
```

Comment:
I added some brief explanation in the comments for the google-gci-dev tab above. I don't want to sound like I am _defining_ what this tab should be used for, especially that I am not entirely sure what people might use it for.